### PR TITLE
Update build instructions and docs

### PR DESCRIPTION
**Pull request by ykoyfman**
Update deploy scripts to point to new OpenLiberty repos, update docs and screenshots.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,7 +102,17 @@ $ minikube ip
 
 With an Ingress controller enabled you can access the app via the IP provided by minikube above.
 
-Now you can use the link **http://[Public IP]** to access your application in a browser.
+If running on IBM Cloud Container Service, you will use the hostname to access the application, which
```
Comment (loafyloaf) [inline README.md]: IBM Cloud Kubertenes Service?

File: docs/build-instructions.md [hunk 2]
```diff
@@ -39,44 +39,44 @@ Install [Docker CLI](https://www.docker.com/community-edition#/download) and a [
 
 Use the following commands to build and push your microservice containers.
 
-> **Note:** For the following steps, you can build and push the images by running
+> **Note:** For the following steps, you can build and push the images by running (requires running get_code.sh in this directory first.)
```
Comment (loafyloaf) [inline docs/build-instructions.md]: maybe add this command into the code snippit?

Comment (ykoyfman) [reply docs/build-instructions.md]: It was already described in a snippet earlier in the doc - so this is just a reminder.

**Comment by loafyloaf** [comment]
also - squash them commits!!

File: docs/build-instructions.md (hunk 2 above)
Comment (loafyloaf) [reply docs/build-instructions.md]: ah.  reads weird, but surely worth mentioning

**Pull request merged**